plugins/github/api.ml

```diff
@@ -341,6 +343,16 @@ module Commit_id = struct
         Some (Astring.String.concat ~sep:"/" branch)
       | _ -> None)
     | `PR _ -> None
+    
+  let pr_fork_branch_name t =
+    match t.id with
+    | `Ref _ -> None
+    | `PR pr -> Some pr.headRefName
+
+  let pr_fork_owner_name t =
```

Comment:
I find the `owner_name` name confusing sometimes. We use `_name` to represent name of something (for instance, `branch_name`, `pr_name`, etc.), and in this function we are trying to use it for `owner/name` representation. How about using something simpler like `pr_fork`?

Reply:
Another option could be `pr_fork_with_owner`?

Reply:
I like `pr_fork_with_owner`! I agree that `owner_name` is not a very good naming convention.